mix.exs

```diff
@@ -46,6 +46,7 @@ defmodule Html5ever.Mixfile do
       files: [
         "lib",
         "native",
+        "priv/native/checksum-*.exs",
```

Comment:
We don't need to put this file in priv, since it is only used during compilation (and priv goes all the way into a release. I would put it in the root of the project.